### PR TITLE
Created initial pytest unit test to validate that training on public dataset

### PR DESCRIPTION
**Pull request by zmhassan**
We are validating that training is successful on public dataset with 2000 log lines.

### CODE REVIEW

**Comment by vpavlin** [comment]
@zmhassan Can you please squash those commits into one? Also is this rebased on `master`?

**Comment by zmhassan** [comment]
@vpavlin when doing the merge I can squash into 1 commit. 

@durandom The PR is ready for review. If you pull this down run `pytest` it will pass

**Comment by vpavlin** [comment]
There is a new error in the Travis build https://travis-ci.org/AICoE/log-anomaly-detector/builds/458806387?utm_source=github_status&utm_medium=notification

File: tests/test_validation_data.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# create a logging format
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+sh = logging.StreamHandler(sys.stdout)
+sh.setFormatter(formatter)
+_LOGGER.addHandler(sh)
+
+w2v_logger = logging.getLogger('gensim.models')
+w2v_logger.setLevel(logging.ERROR)
```
Comment (durandom) [inline tests/test_validation_data.py]: thats a lot of setup, I wonder if this could be done generally in the `Configuration` class. But ok...

**Comment by durandom** [comment]
@zmhassan see https://github.com/AICoE/prometheus-flatliner/pull/26/files for a minimal travis file.
Also see https://github.com/AICoE/log-anomaly-detector/issues/48#issuecomment-441183328 for @fridex comment on the `pipenv install`

File: tests/test_validation_data.py (hunk 1 above)
Comment (zmhassan) [reply tests/test_validation_data.py]: Your right. No need for all this logging setup. I'll strip it down.

**Pull request merged**